vignettes/unit_test_guidance.Rmd

```diff
@@ -103,7 +103,7 @@ For the different calls of your function, consider how the user might apply your
 function and test a variety of possible calls, whilst still remembering the tips
 above that tests should be fast and simple. 
 This is only needed in cases where the complexity and level of flexibility of 
-your function justifies it, e.g. see the test script: https://github.com/pharmaverse/admiral/blob/main/tests/testthat/test-derive_var_extreme_flag.R.
+your function justifies it, e.g. see the test script: https://github.com/pharmaverse/admiral/blob/main/tests/testthat/test-derive_extreme_flag.R.
```

Comment:
This should not be changed because the file was renamed. I.e., after the release `https://github.com/pharmaverse/admiral/blob/main/tests/testthat/test-derive_extreme_flag.R` will be broken.

Reply:
good spot! so this was an intentional broken link that will only work when we release everything to main. i see now. i should have spotted that - will update